### PR TITLE
[AHM-2024] Add 2024 Ahmedabad Event

### PR DESCRIPTION
**Pull request by kaivalya97**
Initial details.

### CODE REVIEW

**Comment by shar1z** [comment]
This looks good to me - I'm just going to ask for a second pair of eyes on this.
Don't forget to send the email to info@ to get your emails added to the distribution list and Slack.

File: data/events/2024-ahmedabad.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  - name: "Mehul Patel"
+    employer: ""
+    twitter: "NomadicMehul"
+    bio: ""
+  - name: "Neel Shah"
+    employer: ""
+    twitter: "shahneel2409"
+    bio: ""
+
+organizer_email: "devopsahmedabad@gmail.com" # Put your organizer email address here
```
Comment (mattstratton) [inline data/events/2024-ahmedabad.yaml:65]: this email address needs to be the `@devopsdays.org` email alias

**Review by mattstratton — CHANGES_REQUESTED** [review]
Please update to use the devopsdays.org email alias for this!

File: data/events/2024-ahmedabad.yaml (hunk 1 above)
Comment (kaivalya97) [reply data/events/2024-ahmedabad.yaml]: I have updated it based on other events but I don't know if this alias is already created or not.

**Comment by mattstratton** [comment]
Have you sent the email to info@devopsdays.org requesting it be created?

**Comment by shar1z** [comment]
@kaivalya97 - this is fine to merge I think - except that you should note that the preferred email address to use is 
ahmedabad@devopsdays.org 
I have created an alias with this email address - in order to get your event launched.
But in your next edit - update the email to the city@devopsdays.org

@mattstratton I believe this is OK to merge

**Review by mattstratton — APPROVED** [review]

**Comment by mattstratton** [comment]
lgtm

**Comment by shar1z** [comment]
@kaivalya97 - this has been merged - feel free to delete the branch.

**Pull request merged**